prototype.py

```diff
@@ -0,0 +1,34 @@
+import random
+#modified code from introduction to computing and programming in python a multimedia approach,
+#third edition, mark j. Guzdial,Barbara Ericson,p.~149
+#Replaces a colour in an image with another image preferably of the same size
+def greenScreen(forGrnd,replaceColour,threshold,backGrnd):
+  for x in range(0,getWidth(forGrnd)):
+    repaint(forGrnd)
+    for y in range(0,getHeight(forGrnd)):
+      forPixel = getPixel(forGrnd,x,y)
+      if distance(getColor(forPixel),replaceColour) <= threshold:
+        setColor(forPixel,getColor(getPixel(backGrnd,x,y)))
+  return forGrnd
+#threshold should be 170 for the cockpit
+##############################################################################################
+
+#purposefully corrupts the image to represent damage taken :::: now with animation
+def glitchScrEffct(image,numBars):
+
+  for index in range(0,numBars):
+    repaint(image)
+    glitchSingleBar(image)
+  return image
+  
+def glitchSingleBar(image):
+    height = getHeight(image)
+    width = getWidth(image)
+    y = random.randint(1,height-1)
+    lftOrRght = random.randint(0,1)                                                                   
+    if lftOrRght == 0:
```

Comment:
Perhaps make a comment on which value represents left and which value represents right. It is quite difficult to work out what is happening here.
Perhaps it can be made clearer using variables.